### PR TITLE
User docs

### PR DESCRIPTION
**Pull request by Martindevelops-songa**

### CODE REVIEW

File: src/Users/controllers/UserControllers.ts [hunk 1]
```diff
@@ -19,7 +19,7 @@ export const CreateUserAccount = async (req: Request, res: Response) => {
 
     const userExists = (await checkUser({ phone, email })) as CheckUserResult;
     if (userExists.userPresent) {
-      res.status(400).json({ message: 'user already exists' });
+      res.status(401).json({ message: 'user already exists' });
```
Comment (dennisngeno-songa) [inline src/Users/controllers/UserControllers.ts:22]: Why are we changing to 401

**Review by dennisngeno-songa — CHANGES_REQUESTED** [review]
Build is failing, it will fail on deployment also if not corretced

File: src/Users/controllers/UserControllers.ts (hunk 1 above)
Comment (Martindevelops-songa) [reply src/Users/controllers/UserControllers.ts:22]: you can create an already existing user.

**Review by Martindevelops-songa — COMMENTED** [review]
done

**Review by dennisngeno-songa — CHANGES_REQUESTED** [review]
Failing build

**Pull request closed without merging**